server/controllers/UsersController.js

```diff
@@ -45,6 +45,52 @@ class UsersController {
           },
         }));
   }
+
+  /**
+    * @description - This method logs in user and return a token.
+    * @param {object} req - The request object bearing the email and password.
+    * @param {object} res - The response object that is returned as json.
+    * @returns {object} - The object with message.
+    * @memberOf UserController
+    * @static
+    */
+  static userLogin(req, res) {
+    const { email, password } = req.body;
+    return User.findOne({
+      where: {
+        email
+      }
+    })
+      .then((userFound) => {
+        if (!userFound || !bcrypt.compareSync(password, userFound.password)) {
+          return res.status(401).json({
+            errors: {
+              message: ['Invalid email or password']
```

Comment:
Why are you using arrays for most of the error messages?

Reply:
The team agreed to follow as stated on the repository readme to display our errors. 
See link [https://github.com/andela/valinor-ah-backend#errors-and-status-codes](url)

Reply:
Let's discuss this during the TTL interaction